backend/src/main/java/de/nordakademie/iaa/noodle/controller/StaticController.java

```diff
@@ -0,0 +1,43 @@
+package de.nordakademie.iaa.noodle.controller;
+
+import io.swagger.annotations.Api;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+
+/**
+ * Home redirection to OpenAPI api documentation
+ */
+@Controller
+@Api(value = "Static")
+public class StaticController {
```

Comment:
I'm not super into this name. Is there a good reason why it's called "StaticController"? Might as well be ApiDocsController or something similar as far as I can tell.

Reply:
I didn't really had a good name for it. I do like `ApiDocsController`. I will change that.